app/assets/stylesheets/pages/_conversations_show.scss

```diff
@@ -1,51 +1,78 @@
-.conversation {
+#conversation-img-title {
   display: flex;
-  flex-direction: row;
-  margin: 60px auto;
+  align-items: baseline;
+  img {
+    border: 3px solid $blue-bg;
+  }
+  p {
+    margin-left: 9px;
+  }
 }
 
-.messages_box {
-  background-color: white;
-  margin-right: 30px;
-  padding: 20px;
+#pal-confirm {
+  display: flex;
+  align-items: baseline;
+  a {
+    margin-right: 8px;
+  }
+}
+
+#request-status {
+  text-align: right;
+  background-color: $light-gray;
+  padding: 7px;
   border-radius: 5px;
-  height: fit-content;
+  position: absolute;
+  right: 14px;
+  top: 14px;
 }
 
-.conversation-header {
-  display: flex;
-  justify-content: space-between;
-  align-items: center;
+#request-avatar {
+  border: 1px solid $blue-bg;
+  position: absolute;
+  right: 14px;
+  top: 14px;
 }
 
-#conversation-img-title {
-  display: flex;
-  align-items: baseline;
+#request-category {
+  background-color: $blue-bg;
+  padding: 10px 15px;
+  border-radius: 5px;
+  text-align: center;
 }
 
-#conversation-img-title img {
-  border: 3px solid $blue-bg;
+.blue {
+  background-color: $blue-bg;
+  border-radius: 6px;
+  padding: 10px;
 }
 
-#conversation-img-title p {
-  margin-left: 9px;
+.not-blue {
+  padding: 10px;
 }
 
-.conversation-messages ul {
-  list-style-type: none;
-  padding-inline-start: 0px;
+.conversation {
+  display: flex;
+  flex-direction: row;
+  margin: 60px auto;
 }
 
+.conversation-header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+
 .conversation-messages {
   height: 300px;
   overflow: auto;
   padding: 15px 50px;
-}
-
-.new_message {
-  display: flex;
-  justify-content: space-between;
-  border-top: 1px solid $light-gray;
+  word-break: break-all;
```

Comment:
I made a PR before seeing that you were tackling the same thing, but I think we want to go with word-wrap instead of word-break (see my PR from this morning). Check this answer out: https://stackoverflow.com/questions/1795109/what-is-the-difference-between-word-break-break-all-versus-word-wrap-break.

Reply:
Thx for the hint, I will change it
to break-all

Reply:
*word-wrap